shared/domains/chelonia/chelonia.js

```diff
@@ -1873,7 +1874,7 @@ function outputEncryptedOrUnencryptedMessage ({
   const signedMessage = innerSigningKeyId
     ? (state._vm.authorizedKeys[innerSigningKeyId] && state._vm.authorizedKeys[innerSigningKeyId]?._notAfterHeight == null)
         ? signedOutgoingData(contractID, innerSigningKeyId, (data: any), this.transientSecretKeys)
-        : signedOutgoingDataWithRawKey(this.transientSecretKeys[innerSigningKeyId], (data: any), this.transientSecretKeys)
+        : signedOutgoingDataWithRawKey(this.transientSecretKeys[innerSigningKeyId], (data: any))
```

Comment:
I'm trying to figure out what the 3rd parameter, `height` was used for, and got stuck here:

```js
      return (additionalData: ?string) => JSON.stringify(this.serialize(additionalData, height))
```

what is `this.serialize`? Is it:

```js
  const serializefn = (additionalData: ?string) => boundStringValueFn(additionalData || '')
```

But that only takes 1 parameter, so is the 3rd parameter `height` completely unused? Why's it there then?

Reply:
There never was a third parameter AFAICT (or, if there was, it has long since been removed). However, `signedOutgoingData` matches the signature for this call, so it likely was a mistake.

As for why there was a `height` to `signedOutgoingDataWithRawKey`, I'm not sure. It was likely left over from a time when I was deciding on the most ergonomic API design. The `height` is, indirectly, used in `serialize` (and by extension `serializefn`) as additional data (well, the entire head is, see `messageToParams`). The `height` is used, directly, for incoming data, for the purpose of determining whether the key is valid.

Reply:
> As for why there was a height to signedOutgoingDataWithRawKey, I'm not sure. It was likely left over from a time when I was deciding on the most ergonomic API design.

Was? It still is there, but it's been renamed to `_height?: number`

It should be removed if it's not used, right? 

> The height is, indirectly, used in serialize (and by extension serializefn) as additional data (well, the entire head is, see messageToParams). The height is used, directly, for incoming data, for the purpose of determining whether the key is valid.

Don't know what you mean... I don't see it being used at all in `signedOutgoingDataWithRawKey`

Reply:
> Was? It still is there, but it's been renamed to _height?: number
>
> It should be removed if it's not used, right?

I meant in the returned object. Yes, probably it should be removed (`_height`).

> Don't know what you mean... I don't see it being used at all in signedOutgoingDataWithRawKey

Like I said, it's being used _indirectly_ as `additionalData`. This is done in `messageToParams`. `height`/`_height` are not used at all here.